### PR TITLE
Update for Tri 2 in 2021

### PR DESCRIPTION
**Pull request by jaredboult**
I think that's all :)

### CODE REVIEW

File: src/deadify.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (() => {
 	function getWeek() {
 		// The uni week without factoring out holidays/breaks
-		let weekNum = getWeekOfYear(new Date()) - 7;
+		let weekNum = getWeekOfYear(new Date()) - 26;
 
 		// Mid-term break
 		if (weekNum > 6 && weekNum < 9) {
```
Comment (NickGeek) [inline src/deadify.js:7]: You also need to adjust this for the mid tri break.

**Review by NickGeek — CHANGES_REQUESTED** [review]

File: src/deadify.js (hunk 1 above)
Comment (jaredboult) [reply src/deadify.js:7]: The T2 mid-term break is two weeks long when weekNum is 7 and 8, same as T1 mid-term break, so I'm not sure it needs adjusting this time?

Comment (NickGeek) [reply src/deadify.js:7]: Sweet

**Comment by NickGeek** [comment]
Just one more thing, for caching reasons can you please edit the deadify.js filename (and the script url in the html). Renaming to something like `deadify.2021t2.js` should work. Thanks!

**Comment by jaredboult** [comment]
Sure thing, have renamed it

**Review by NickGeek — APPROVED** [review]

**Pull request merged**